src/dnx.coreclr/dnx.coreclr.cpp

```diff
@@ -506,7 +506,7 @@ extern "C" HRESULT __stdcall CallApplicationMain(PCALL_APPLICATION_MAIN_DATA dat
 
     if (FAILED(hr))
     {
-        wprintf_s(L"TPA      %d %s\n", wcslen(pwszTrustedPlatformAssemblies), pwszTrustedPlatformAssemblies);
+        wprintf_s(L"TPA      %Iu %s\n", wcslen(pwszTrustedPlatformAssemblies), pwszTrustedPlatformAssemblies);
```

Comment:
Looks like we don't compile this file on Linux/mac, which needs `dnx.coreclr.unix/dnx.coreclr.cpp `. This change should be exclusively for windows.
(BTW, Travis CI passes below)